wpigui/src/main/native/cpp/portable-file-dialogs.cpp

```diff
@@ -378,19 +378,6 @@ std::string path::home()
     auto user_profile = internal::getenv("USERPROFILE");
     if (user_profile.size() > 0)
         return user_profile;
-    // Otherwise, try GetUserProfileDirectory()
```

Comment:
What was the build issue that keeps this from working?

Reply:
Oh, you just need to add `#pragma comment(lib, "Advapi32.lib")` to the file.

Reply:
Should that be wrapped in any compiler type ifs or is it ok being alone?